tests/testthat/test-tait-bryan.R

```diff
@@ -1,7 +1,17 @@
-test_that("unspecified unit argument throws error", {
-  expect_error(tait_bryan(yaw = 0, pitch = 0, roll - 0))
+test_that("missing unit argument throws error", {
+  expect_error(tait_bryan(yaw = 0, pitch = 0, roll = 0))
 })
 
+test_that("unexpected unit argument throws error", {
+  # match.arg should not throw error for degree
+  expect_equal(
+    tait_bryan(yaw = 0, pitch = 0, roll = 0, unit="degree"),
+    quat(1, 0, 0, 0)
```

Comment:
This is the identity quaternion, so it will work for both degrees and radians

Try with `pitch=60` and the result being `cos(pi/3), sin(pi/3), 0, 0`